administrator/language/de-DE/com_installer.ini

```
@@ -182,10 +182,10 @@ COM_INSTALLER_MSG_WARNINGFURTHERINFO="Weitere Informationen zu den Warnungen"
 COM_INSTALLER_MSG_WARNINGFURTHERINFODESC="Weitere Informationen zu den Warnungen können in der <a href='https://docs.joomla.org/Special:MyLanguage/Help4.x:Information:_Warnings' class='alert-link' target='_blank' rel='noopener noreferrer'>Joomla-Dokumentation</a> nachgelesen werden."
 COM_INSTALLER_MSG_WARNINGS_FILEUPLOADISDISABLEDDESC="Das Hochladen von Dateien wird aber benötigt, um Erweiterungen über den Installer zu installieren."
 COM_INSTALLER_MSG_WARNINGS_FILEUPLOADSDISABLED="Das Hochladen von Dateien ist deaktiviert"
-COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTSET="Das temporäre Verzeichnis für Joomla! ist nicht gesetzt"
-COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTSETDESC="Im temporären Verzeichnis speichert Joomla! hochgeladene Pakete, entpackt sie dort und kopiert die Dateien in die entsprechenden Verzeichnisse. Wenn der Pfad zu diesem Verzeichnis nicht in der „configuration.php“ definiert ist („$tmp_path“), kann man keine Paketdateien hochladen und installieren. Das Verzeichnis muss vorhanden und durch Joomla! beschreibbar sein."
-COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTWRITEABLE="Das temporäre Verzeichnis für Joomla! existiert nicht oder ist schreibgeschützt"
-COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTWRITEABLEDESC="Das in Joomla! gesetzte temporäre Verzeichnis ist schreibgeschützt oder existiert nicht. Dies kann zu Problemen beim Hochladen von Erweiterungen in Joomla! führen. Wenn es Probleme beim Hochladen von Erweiterungen gibt, sollte das in der „configuration.php“ gesetzte Hochladeverzeichnis („%s“) überprüft werden und ggf. der Schreibschutz aufgehoben werden."
+COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTSET="Das temporäre Verzeichnis für Joomla ist nicht gesetzt"
+COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTSETDESC="Im temporären Verzeichnis speichert Joomla hochgeladene Pakete, entpackt sie dort und kopiert die Dateien in die entsprechenden Verzeichnisse. Wenn der Pfad zu diesem Verzeichnis nicht in der „configuration.php“ definiert ist („$tmp_path“), kann man keine Paketdateien hochladen und installieren. Das Verzeichnis muss vorhanden und durch Joomla beschreibbar sein."
```

```suggestion
COM_INSTALLER_MSG_WARNINGS_JOOMLATMPNOTSETDESC="Im temporären Verzeichnis speichert Joomla! hochgeladene Pakete, entpackt sie dort und kopiert die Dateien in die entsprechenden Verzeichnisse. Wenn der Pfad zu diesem Verzeichnis nicht in der „configuration.php“ definiert ist („$tmp_path“), kann man keine Paketdateien hochladen und installieren. Das Verzeichnis muss vorhanden und durch Joomla! beschreibbar sein."
```
see Comment on line R97 file administrator/language/de-DE/com_config.ini